code/datums/elements/alloy_regen.dm

```diff
@@ -0,0 +1,84 @@
+/** Object integrity regeneration element added by alien alloy.
+  */
+/datum/element/alloy_regen
```

Comment:
Is there any reason this element has to be alloy specific?

Reply:
Probably not, I'll rename it.

Reply:
Done.